UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Services/FssService.cs

```diff
@@ -312,8 +341,6 @@ private AddFileToBatchRequestModel CreateAddFileRequestModel()
             {
                 Attributes = new List<KeyValuePair<string, string>>()
                 {
-                    new KeyValuePair<string, string>("Exchange Set Type", "Base"),
-                    new KeyValuePair<string, string>("Media Type", "DVD"),
                     new KeyValuePair<string, string>("Product Type", "AVCS")
```

Comment:
This can be simplified as follows:

                Attributes = new List<KeyValuePair<string, string>>()
                {
                    new("Product Type", "AVCS")
                }